### ISSUE TITLE
Documenting development progress

### ISSUE THREAD

**Issue by ekaterinaaleksee** [amosproj/amos2022ss03-turtlebot-fleet-management#29]
Should SICK Extranet be used and everyone agrees on it? Does it help to make the process more transparent as well as easier?

**Comment by UmangBR** [comment]
Yes.  Documenting along with progression in work is more organized and helps avoid long threads of Q&A on Slack.  It is easier not only for the industry partners and product owners to keep track of work, but can also help SDs to minimize confusion betwwen themselves. GitHub feature board does keep track of all tasks but each issue can only have short descriptions. Commenting on GitHub issues is an option but again it's Sick extranet would be more organized.
